nginx/nginx-plus-router-mesh.conf.j2

```diff
@@ -0,0 +1,68 @@
+{% if log_to_syslog %}
+error_log {{ syslog_error_log_location }};
+{% else %}
+error_log {{ error_log_location }};
+{% endif %}
+
+worker_processes  auto;
+
+events {
+  worker_connections  1024;
+}
+
+daemon off;
+
+http {
+
+  include mime.types;
+  include nginx-gz.conf;
+
+  client_max_body_size 30M;
+
+    {% if log_to_syslog %}
+        log_format json '{"service_name": "{{ service_log_name }}",'
+             '"time": "$time_iso8601", '
+             '"remote_addr": "$remote_addr", '
+             '"remote_user": "$remote_user", '
+             '"body_bytes_sent": $body_bytes_sent, '
+             '"request_time": $request_time, '
+             '"status": $status, '
+             '"request": "$request", '
+             '"request_method": "$request_method", '
+             '"http_referrer": "$http_referer", '
+             '"http_x_forwarded_for": "$http_x_forwarded_for", '
+             '"http_user_agent": "$http_user_agent"}';
+    {% else %}
+        log_format main {{ service_log_name }} '$remote_addr - $remote_user [$time_local] "$request" '
+              '$status $body_bytes_sent "$http_referer" '
+              '"$http_user_agent" "$http_x_forwarded_for"';
+    {% endif %}
+
+    {% if log_to_syslog %}
+        access_log {{ syslog_access_log_location }};
+    {% else %}
+        access_log {{ access_log_location }};
+    {% endif %}
+
+  server {
+    listen       80;
+
+    keepalive_timeout	3600s;
+    keepalive_disable	none;
+    keepalive_requests  100000;
+
+    server_name {{ name }};
+
+    root {{ root }};
+
+    include default-location.conf;
+
+    location = /status.html {
```

Comment:
The status API is going to be deprecated in favor of the new unified API and we should use that going forward.

Please update the config using the transition guide: https://www.nginx.com/blog/transitioning-to-nginx-plus-api-configuration-monitoring/

Reply:
Definitely, I'll change this.